src/Validator/hdtAssetValidator.cpp

```
@@ -518,60 +518,88 @@ namespace hdt
+
+		// The single legacy file first, then the drop-in folder in ascending filename order — the same
+		// order the runtime loader feeds them (so the report reflects the effective mappings).
+		fs::path bbpFile = "data/SKSE/Plugins/hdtSkinnedMeshConfigs/defaultBBPs.xml";
+		if (fs::exists(bbpFile, ec))
+			parseDoc(bbpFile);
+		else
+			logger::info("[Validator] defaultBBPs.xml not found at {}", PathToUtf8(bbpFile));
+
+		fs::path folder = "data/SKSE/Plugins/hdtSkinnedMeshConfigs/defaultBBPs";
+		if (fs::is_directory(folder, ec)) {
+			std::vector<fs::path> files;
```

Clang (14.0.6)

[warning] variable 'files' is not initialized

(cppcoreguidelines-init-variables)